IntegrationTestSuite/WebservicesIntegrationTests/Tests/EngineeringModel/Iteration/IterationTestFixture.cs

```diff
@@ -97,6 +97,23 @@ public void VerifyThatFrozenIterationDoesNotAcceptModelChanges()
             postBody = base.GetJsonFromFile(postBodyPath);
 
             Assert.That(() => this.WebClient.PostDto(iterationUri, postBody), Throws.TypeOf<WebException>());
+
+            var checkError = false;
+
+            try
+            {
+                this.WebClient.PostDto(iterationUri, postBody);
+            }
+            catch (Exception ex)
+            {
+                checkError = true;
+                var webException = ex as WebException;
+                Assert.That(webException, Is.Not.Null);
```

Comment:
You could surround the 3 assert.That by Assert.Multiple(() => {...}); to get all results even if the first one fails

Reply:
Yes, I should remember this... Not in my system yet.